packages/core/src/plugins/QueueFlushingPlugin.ts

```diff
@@ -75,12 +76,30 @@ export class QueueFlushingPlugin extends UtilityPlugin {
     // Wait for the queue to be restored
     try {
       await this.isRestored;
+
+      if (this.timeoutWarned === true) {
```

Comment:
I guess this statement is trying to prevent the timeout message from keep popping up? two questions (forgive me if they are naive):
1. not sure how `this.isRestored` works, but looks the catch block is intended for`this.isRestored`. so if we have this if statement after the await, will it ever be triggered? because if there is an exception, it goes straight to the catch block. 
2. should we have `timeoutWarned` be reset to false somewhere? maybe when the flush actually succeed? otherwise the value may not be true for some cases.

Reply:
not at all! 
1. yes it will be triggered on a successfully resolved promise of `this.isRestored`
2. good call- added it